EIPS/eip-wrapped_deposit.md

```diff
@@ -0,0 +1,137 @@
+---
+eip: <to be assigned>
+title: Wrapped Deposits
+description: A singleton contract for managing asset deposits.
+author: Justice Hudson (@jchancehud)
+discussions-to: https://ethereum-magicians.org/t/wrapped-deposit-contract-eip/7740
+status: Draft
+type: Informational
```

Comment:
```suggestion
type: Standards Track
catrgory: ERC
```
This is an application-level standard, so it would belong in ERC (as opposed to Informational)